Rotate kubelet client certificates GA, CSR API GA (#21108)
kubernetes/website: content/en/docs/tasks/tls/managing-tls-in-a-cluster.md

Notice that the `server.csr` file created in step 1 is base64 encoded
and stashed in the `.spec.request` field. We are also requesting a
certificate with the "digital signature", "key encipherment", and "server
auth" key usages, signed by the `kubernetes.io/kubelet-serving` signer.

> We support all key usages and extended key usages listed
> [here](https://godoc.org/k8s.io/api/certificates/v1beta1#KeyUsage)

Reviewer: Is it correct to revise to:
```suggestion
Kubernetes supports all key usages and extended key usages listed in
[`KeyUsage`](https://godoc.org/k8s.io/api/certificates/v1#KeyUsage)
```
?
Suggesting this mainly to [avoid using “we”](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we) but also because I assume that KeyUsage is also going to stabilize into `k8s.io/api/certificates/v1`.

Author: I actually dropped that whole sentence in favor of pointing at the signer documentation, which call out the expected usages
